doc/git-subrepo.swim

```diff
@@ -599,6 +599,74 @@ If you want to chat about the `git-subrepo` command, join `#gitcommands` on
 * Written in (very modern) Bash, with full test suite. Take a look.
 * A `.gitrepo` file never is in the top level dir (next to a `.git/` dir).
 
+= Working with nested subrepos
+
+`git-subrepo` supports adding subrepos which themselves make use of
+subrepos. In fact, subrepo itself is built this way. Take a look in the
+`ext/` folder, it houses 2 dependencies as subrepos, `test-more-bash` and
+`bashplus`. 
+
+`test-more-bash` itself depends on 2 more subrepos: `bashplus` again, and
+`test-tap-bash`.
+
+The structure is therefore as follows:
+
+  git-subrepo
+    |- bashplus
+    |- test-more-bash
+      |- bashplus
+      |- test-tap-bash
+
+However, it's important to understand how this works. Here's the key idea:
+*`git-subrepo` is 100% agnostic about your dependencies!*
+
+There is *no* special handling for nested subrepos. When you clone a
+subrepo, *all* `git-subrepo` does is download the code and set up the subrepo
+file. It couldn't care less about the actual contents of your code.
```

Comment:
"It couldn't care less about the actual contents of your code." Feels both a little harsh and a lot personified. Could this be removed and the following sentence reworked that I feel describes what is happening quite well?